### PR TITLE
Add multi-streaming example

### PR DESCRIPTION
**Pull request by andressbarajas**

### CODE REVIEW

**Comment by DC-SWAT** [comment]
Maybe use ADPCM + PCM? For a change.

File: examples/dreamcast/sound/multi-stream/main.c [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    snprintf(volume_str, 32, "Brushing Volume: %3i", brushing_vol);
+    bfont_draw_str(vram_s + y*640+x, 640, color, volume_str);
+
+    y += 48;
+    bfont_draw_str(vram_s + y*640+x, 640, color, "Press Start to exit program");
+}
+
+static cont_state_t* get_cont_state() {
+    maple_device_t* cont;
+    cont_state_t* state;
```
Comment (ljsebald) [inline examples/dreamcast/sound/multi-stream/main.c:134]:
```suggestion
static cont_state_t *get_cont_state() {
    maple_device_t *cont;
    cont_state_t *state;
```

File: examples/dreamcast/sound/multi-stream/main.c [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+    cont = maple_enum_type(0, MAPLE_FUNC_CONTROLLER);
+    if(cont) {
+        state = (cont_state_t*)maple_dev_status(cont);
```
Comment (ljsebald) [inline examples/dreamcast/sound/multi-stream/main.c:138]:
```suggestion
        state = (cont_state_t *)maple_dev_status(cont);
```

File: examples/dreamcast/sound/multi-stream/main.c [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+    wav_stream_hnd_t brushing = wav_create("/rd/brushing.wav", LOOP);
+    wav_volume(brushing, brushing_volume);
+
+    wav_play(faucet);
+    wav_play(brushing);
+
+    uint32_t current_buttons = 0;
+    uint32_t changed_buttons = 0;
+    uint32_t previous_buttons = 0;
```
Comment (ljsebald) [inline examples/dreamcast/sound/multi-stream/main.c:42]:
```suggestion
```

File: examples/dreamcast/sound/multi-stream/main.c [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+static void draw_instructions(int faucet_vol, int brushing_vol);
+
+static cont_state_t* get_cont_state();
+static int button_pressed(uint32_t current_buttons, uint32_t changed_buttons, uint32_t button);
+
+int main(int argc, char **argv) {
+    uint8_t faucet_volume = 240;
+    uint8_t brushing_volume = 240;
+    int volume_changed = 1;
+    cont_state_t* cond;
```
Comment (ljsebald) [inline examples/dreamcast/sound/multi-stream/main.c:24]:
```suggestion
    cont_state_t *cond;
    uint32_t current_buttons = 0;
    uint32_t changed_buttons = 0;
    uint32_t previous_buttons = 0;
```

File: examples/dreamcast/sound/multi-stream/main.c [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+*/
+
+#include <kos.h>
+#include <wav/sndwav.h>
+
+#define LOOP 1
+
+static void draw_instructions(int faucet_vol, int brushing_vol);
+
+static cont_state_t* get_cont_state();
```
Comment (ljsebald) [inline examples/dreamcast/sound/multi-stream/main.c:17]:
```suggestion
static cont_state_t *get_cont_state();
```

**Review by ljsebald — CHANGES_REQUESTED** [review]

**Comment by andressbarajas** [comment]
DONE

**Review by ljsebald — DISMISSED** [review]

**Review by ljsebald — APPROVED** [review]

**Pull request merged**